### PR TITLE
Don't show negative numbers in progress bar

### PR DESCRIPTION
**Pull request by alanwguo**
Signed-off-by: Alan Guo <aguo@anyscale.com>

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by ericl — APPROVED** [review]
LGTM. Do you want to make a separate PR adding this to the Grafana query as well for tasks in the Metrics tab?

File: dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum(ray_tasks) by (State)",
+          "expr": "sum(max_over_time(ray_tasks{State=~\"FINISHED|FAILED\"}[14d])) by (State) or clamp_min(sum(ray_tasks{State!~\"FINISHED|FAILED\"}) by (State), 0)",
```
Comment (rkooo567) [inline dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json:90]: is this relevant to this PR?

Comment (alanwguo) [reply dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json:90]: yes, we need to update the grafana task graph to account for the fact that we lose data after a worker exits.

**Review by rickyyx — APPROVED** [review]
<div><br/>via <a href="https://giphy.com/gifs/eating-mrw-yyZRSvISN1vvW">GIPHY</a></div>
